scripts/generate_tiny_models.py

```diff
@@ -209,6 +208,17 @@ def init_weights_tiny_model(model):
     init_weights_tiny_model(model)
     push_to_hub(model, tokenizer, "tiny", suffix)
 
+# Special case for databricks/dbrx-instruct as it requires specific changes in the config
+model_id = "databricks/dbrx-instruct"
+tokenizer = AutoTokenizer.from_pretrained(model_id)
+config = DbrxConfig.from_pretrained(model_id, n_layers=2, n_heads=4, d_model=24)
```

Comment:
perhaps we can also reduce `vocab_size` as its huge.. but it might break things?

Reply:
Yes, I think we cannot touch the vocab size, it's assumed everywhere the the tokenizer vocab size equals the model vocab size